src/components/Reports/ReportsHoursVisits/ReportsHoursVisits.js

```diff
@@ -0,0 +1,138 @@
+        periodBy: 'hours',
+      });
+      if (!hoursVisitsdata.success) {
+        setState({ loading: false });
+      } else {
+        const visitsByWeekDayAndHour = hoursVisitsdata.value.reduce((accumulator, item) => {
+          const weekDay = item.from.getUTCDay();
+          const hour = item.from.getUTCHours();
+          accumulator[weekDay][hour] += item.quantity;
+          return accumulator;
+        }, createEmptyWeekDayHoursMatrix());
+        setState({
+          loading: false,
+          visits: visitsByWeekDayAndHour,
+        });
+      }
+    };
+
+    fetchData();
+  }, [datahubClient, dateFrom, domainName]);
+
+  return (
+    <div className="wrapper-reports-box">
+      <div className="reports-box" style={{ width: '900px' }}>
+        <S.Header>
+          <small className="title-reports-box">
+            <FormattedMessage id="reports_hours_visits.title" />
+          </small>
+          <div>
+            <div>
+              <S.Circle />
+              <p>
+                <FormattedMessage id="reports_hours_visits.few_visits" />
+              </p>
+            </div>
+            <div>
+              <S.Circle medium />
+              <p>
+                <FormattedMessage id="reports_hours_visits.medium_visits" />
+              </p>
+            </div>
+            <div>
+              <S.Circle big />
+              <p>
+                <FormattedMessage id="reports_hours_visits.lot_visits" />
+              </p>
+            </div>
+          </div>
+        </S.Header>
+
+        {state.loading ? (
+          <Loading />
+        ) : state.visits ? (
+          <S.List>
+            {state.visits.map((weekDays, weekDayIndex) => (
+              <S.Row key={weekDayIndex}>
+                <div className="weekday">
+                  <p>
+                    <FormatWeekDayIndex value={weekDayIndex} format={'short'} />
+                  </p>
+                  <S.Dash>- - -</S.Dash>
+                </div>
+
+                {weekDays.map((quantity, hour) => (
+                  <S.Column key={'' + weekDayIndex + '' + hour}>
+                    {quantity <= 300 ? (
+                      <>
+                        <S.Dash>-</S.Dash>
+                        <S.Circle />
+                        <S.Dash>-</S.Dash>
+                      </>
+                    ) : quantity <= 600 ? (
+                      <S.Circle medium />
+                    ) : (
+                      <S.Circle big />
+                    )}
+                    <S.Tooltip>
+                      <p>
+                        <FormatWeekDayIndex value={weekDayIndex} format={'long'} />
+                        <span>{hour}h</span>
+                      </p>
+                      <strong>
+                        <FormattedMessage id="reports_hours_visits.users" /> {quantity}
+                      </strong>
+                    </S.Tooltip>
+                  </S.Column>
+                ))}
+                <S.Dash>- - -</S.Dash>
```

Comment:
The content of `<S.Dash>` will be always ` - `?

Reply:
Yes

Reply:
In that case, I prefer having a component with the styles and content.

Or, having a more general style than Dash and reuse it in other places.

Reply:
i don't know if can reuse because has specific color